src/chapters/booleans-and-conditionals/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Goal Setting and Getting into the Right Mindset
+=======================================================
+
+During this studio, we will ask you to think about your mindset when it comes to facing challenges.
+Do challenges deter you or delight you?
+If challenges are something you delight in, you might have a growth mindset.
+A **growth mindset** can be powered by one word, “yet”.
+The word “yet” allows us to acknowledge that we don’t know something AND that we are capable of learning it.
+For example, with “yet”, the sentence “I don’t understand booleans” becomes “I don’t understand booleans `yet`.”
+
+Growing as developers is something that will take our whole careers.
```

Comment:
In this sentence and the 4 that follow, you used the word "developer(s)" quite a bit.  Let's try to reduce some of the instances...
For the first sentence, how about, "Growth must occur over an entire career."

Reply:
Thanks!